python/nano/src/bigdl/nano/automl/pytorch/hposearcher.py

```diff
@@ -124,6 +114,8 @@ def search(self,
                resume=False,
                target_metric=None,
                n_parallels=1,
+               auto_optimize=False,
```

Comment:
leave `auto_optimize=True`. it will be set to False by default if "latency" is not in target_metric

Reply:
If leave auto_optimize=True, we suppose if user use "latency" it will automatically trigger optimizations and of course it will be slower.Does It really should be a default option?